src/components/CheckoutModal/CheckoutModal.tsx

```diff
@@ -65,7 +67,7 @@ class CheckoutModal extends React.Component<Props, State> {
         alertRateLimitError(error)
         return
       } else {
-        safeAlert('Something went wrong. Please check your internet connection.')
+        safeAlert(constants.errors.error_429)
```

Comment:
Instead of calling this `error_429` I'd recommend calling it `constants.errors.somethingWentWrong`. It's a very generic error message that might get reused elsewhere.